lib/bloc/todo_bloc.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:equatable/equatable.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../models/item.dart';
+
+part 'todo_event.dart';
```

Comment:
@LuchoTurtle we need to phase out the "Todo" naming and namespacing.
We aren't building a "Todo" App. 
It's fine to think about it that way if you're explaining it to someone who hasn't read the [Roadmap](https://github.com/dwyl/product-roadmap) but our App will do a _lot_ more than be a Todo list.
There are plenty of those on the App Store and another one isn't needed.

So in a future PR we need to remove `todo_` or `TodoBloc` and just call it `app_` or `AppBloc`.